appshell/appshell_extensions.js

```diff
@@ -829,6 +859,20 @@ if (!appshell.app) {
     appshell.app.setZoomLevel = function (zoomLevel, callback) {
         SetZoomLevel(callback || _dummyCallback, zoomLevel);
     };
+
+    /**
+     * Install command line scripts to make Brackets launchable from command line.
+     * Right now usage is restricted to MAC only.
+     *
+     * @param {number}
+     *
+     * @return int. The remote debugging port used by the appshell.
+     */
+     native function InstallCommandLineTools();
```

Comment:
Should this be called something Brackets specific like `InstallCommandLineToolsForBrackets` or `AddBracketsToCommandLine` 

This may just be me, but I am think 'xcode' command line tools by default when I hear command line tools :) Please feel free to ignore.